health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/appConfigHelpers.js

```
@@ -163,18 +214,44 @@ export const restructure = (data1, fieldTypeMasterData = [], parent) => {
 // Update reverseRestructure to use getTypeAndFormatFromAppType
 export const reverseRestructure = (updatedData, fieldTypeMasterData = []) => {
   return updatedData.map((section, index) => {
     const properties = section.cards?.[0]?.fields.map((field, fieldIndex) => {
       const typeAndFormat = getTypeAndFormatFromAppType(field, fieldTypeMasterData);
-      const validations = addValidationArrayToConfig(field, fieldTypeMasterData);
+      const toArrayFields = addToArrayFields(field, fieldTypeMasterData);
       return {
         label: field.label || "",
         order: fieldIndex + 1,
         value: field.value || "",
         hidden: false, // can't be derived from updatedData unless explicitly added
```

Biome (1.9.4)

[error] This property value named hidden is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named hidden


(lint/suspicious/noDuplicateObjectKeys)